tests/core/test_actions.py

```diff
@@ -2665,6 +2665,65 @@ async def test_action_extract_slots_non_required_form_slot_with_from_entity_mapp
     )
     assert events == [SlotSet("form1_info1", "info1"), SlotSet("form1_slot1", "Filled")]
 
+async def test_action_multiple_intents_same_values():
```

Comment:
This is a good test to have, we also need a unit test in `tests/shared/core/test_slot_mappings.py` that `check_mapping_validity` method gives the correct result in case of valid and invalid inputs.